ui/app/AppLayouts/Chat/controls/community/EnsPanel.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls.Validators 0.1
+
+ColumnLayout {
+    id: root
+
+    enum EnsType {
+        Any,
+        CustomSubdomain
+    }
+
+    property int ensType: EnsPanel.EnsType.Any
+    property alias domainName: domainNameInput.text
+    property alias domainNameValid: domainNameInput.valid
+
+    spacing: 0
+
+    QtObject {
+        id: d
+
+        // values from design
+        readonly property int pickerHeight: 36
+        readonly property int pickerFontSize: 13
+        readonly property int pickerLeftPadding: 12
+        readonly property int pickerRightPadding: 9
+    }
+
+    // TODO (>=5.15): use inline components to reduce code duplication
+    StatusListItem {
+        title: qsTr("Any domain")
+
+        Layout.fillWidth: true
+        Layout.preferredHeight: d.pickerHeight
+
+        leftPadding: d.pickerLeftPadding
+        rightPadding: d.pickerRightPadding
+        statusListItemTitle.font.pixelSize: d.pickerFontSize
+
+        components: [
+            StatusRadioButton {
+                checked: root.ensType === EnsPanel.EnsType.Any
+                size: StatusRadioButton.Size.Small
+            }
+        ]
+
+        MouseArea {
```

Comment:
It's done this way to hide MouseAreas from both StatusListItem and StatusRadioButton. Otherwise StatusRadioButton would intercept the mouse event when clicking on the radio button ring.

Reply:
Ah yes, the joy of overlapping MouseAreas